app/src/features/apiClient/APIClientManager/APIClientManager.ts

```diff
@@ -0,0 +1,46 @@
+import { makeRequest } from "../screens/apiClient/utils";
```

Comment:
Can we refractor the folder in the following folder structure. This will let us add more modules in future based on appMode (scripts modules for desktop app and extension can be different)


```
- features/apiClient
    - helpers
        - apiClientManager/
            - modules/
                - variables/
                - scripts/
             - index.ts 
    - screens
```